### PR TITLE
Welcome email

### PR DESCRIPTION
**Pull request by kallenilver**
Lets start sending a welcome email to people who just registered!

And while we are at it, let's change from boring info-default adress to fun and games hello-adress!

This PR is probably best viewed commit by commit.

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -872,6 +872,18 @@ en:
...
+      regardless_of_how_much: Regardless of how much you currently know and already do for the climate, we hope that by joining our community you will become even more inspired and learn more about how we can reduce our emissions together. For inspiration and more information, please follow us on %{facebook}, %{instagram}, %{twitter} and check out our %{blog}.
+      we_are_many: There are many of us who want to do good for the climate, and we believe that coming together as a community is important in order to succeed. To keep the community active, we regularly invite you to meetups and climate strikes, to talk about how we can take climate action together. Invitations to events and information on our progress will be shared with you in our newsletter, 4-6 times per year.
+      if_you_have_any_questions: If you have any questions or just want to say hello, feel free to drop me a line at tove@goclimateneutral.org!
+      thank_you: Thank you for contributing to a more sustainable future!
+      best_regards: Warm regards,
+      the_gcn_team: Tove from GoClimateNeutral
```
Comment (stefanlindbohm) [inline config/locales/en.yml:886]: Is this what we decided? I.e. that we sign with Tove's name?

File: config/locales/en.yml [hunk 2]
```diff
@@ -872,6 +872,18 @@ en:
         thank_you: Thank you for calculating your climate footprint!
         link_description: "To make it easier to share your results with your team and get back to them later, here's a link to the results:"
         reply_to_this_email: If you have any questions, just reply to this email. We're happy to help!
+    welcome:
+      from: Tove from GoClimateNeutral
+      subject: Welcome to GoClimateNeutral
+      welcome_to_gcn: Hello!
+      big_thanks: A big thank you for joining us and contributing to a more sustainable world!
+      you_now_support: You now support certified climate projects that also contribute to other types of sustainable development around the world. Well done!
```
Comment (stefanlindbohm) [inline config/locales/en.yml:880]: Maybe you already did iterations on the copy, but isn't "also contribute to other types of sustainable development" a bit vague?

File: config/locales/en.yml [hunk 3]
```diff
@@ -872,6 +872,18 @@ en:
...
+      regardless_of_how_much: Regardless of how much you currently know and already do for the climate, we hope that by joining our community you will become even more inspired and learn more about how we can reduce our emissions together. For inspiration and more information, please follow us on %{facebook}, %{instagram}, %{twitter} and check out our %{blog}.
```
Comment (stefanlindbohm) [inline config/locales/en.yml:881]: Maybe "regardless of where you are on your journey towards a fully sustainable life"? Current wording makes it seem a little condescending to me, as in "maybe you know very little".

File: config/locales/en.yml (hunk 1 above)
Comment (kallenilver) [reply config/locales/en.yml:886]: This is what we ended up with, yes :) Me, Cissi and Tove found it feeling more personal this way.

File: config/locales/en.yml (hunk 2 above)
Comment (kallenilver) [reply config/locales/en.yml:880]: Agree! We couldn't come up with anything better though and deemed it good enough for now.

File: config/locales/en.yml (hunk 3 above)
Comment (kallenilver) [reply config/locales/en.yml:881]: Great, I'll fix!

**Pull request merged**